src/main/java/de/retest/recheck/cli/subcommands/Stage.java

```diff
@@ -0,0 +1,44 @@
+package de.retest.recheck.cli.subcommands;
+
+import java.util.Arrays;
+import java.util.List;
+
+//import de.retest.recheck.ChangeApplier;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+import picocli.CommandLine.Parameters;
+
+@Command( name = "stage", description = "Stage all changes of a certain state and all same changes to other states" )
```

Comment:
Descriptions should end with consistently, i.e. with or without a period (compare e.g. `description` below).

Reply:
How do you get the parameter description to show?